#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_sign_nunit_util.dpatch by  <jelmer@samba.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix dir seperator for POSIX

@DPATCH@

--- nunit2.2-2.2.0/util/AssemblyInfo.cs	2004-08-06 18:35:12.000000000 +0200
+++ nunit2.2-2.2.0-new/util/AssemblyInfo.cs	2006-04-26 00:34:28.000000000 +0200
@@ -82,6 +82,9 @@
 //   (*) Delay Signing is an advanced option - see the Microsoft .NET Framework
 //       documentation for more information on this.
 //
+#if !StronglyNamedAssembly
 [assembly: AssemblyDelaySign(false)]
-[assembly: AssemblyKeyFile("")]
+#else
+[assembly: AssemblyKeyFile("../../../nunit.key")]
 [assembly: AssemblyKeyName("")]
+#endif
--- nunit2.2-2.2.0/util/nunit.util.build	2004-07-19 15:23:06.000000000 +0200
+++ nunit2.2-2.2.0-new/util/nunit.util.build	2006-04-26 00:34:50.000000000 +0200
@@ -12,7 +12,7 @@
 
     <target name="build" depends="copy">
         <csc target="library" output="${bin.dir}/nunit.util.dll"
-                debug="${build.debug}" define="${build.defines.csc}">
+                debug="${build.debug}" define="${build.defines.csc},StronglyNamedAssembly">
             <sources basedir="${src.dir}/util">
                 <include name="**/*.cs"/>
             </sources>
